### PR TITLE
chore: Prepare 0.6.0 release

### LINKED ISSUE #1208: Add docs for `ansi_stripper` transform
Functionality was added in #1188, so we should add it to the documentation.

### LINKED ISSUE #1214: Document geoip transform
Implemented in #1015

### PR DESCRIPTION
**Pull request by binarylogic**
This prepares Vector for the `0.6.0` release. As a result, I needed to finish documentation for the new `geoip` and `ansi_stripper` transforms that were not documented hence why this closes issues.

Closes #1214 
Closes #1208

### CODE REVIEW

**Comment by a-rodin** [comment]
I have one thought: does the user visiting [downloads page](https://deploy-preview-1274--vector-project.netlify.com/download) really need to know about MUSL or MSVC? Maybe we can shorten the downloads names to just `Linux (x86_64)` and `Windows (x86_64, 7+)`?

File: .meta/transforms/geoip.toml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+[transforms.geoip]
+allow_you_to_description = "enrich events with geolocation data from the MaxMind GeoIP2 and GeoLite2 databases"
+common = false
```
Comment (amitsaha) [inline .meta/transforms/geoip.toml]: I think we should mention here that we only recognize city databases here?

File: .meta/transforms/geoip.toml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+function_category = "enrich"
+guides = []
+input_types = ["log"]
+output_types = ["log"]
+resources = []
+
+[transforms.geoip.options.database]
+type = "string"
+common = true
+examples = ["/path/to/GeoLite2-City.mmdb", "/path/to/GeoLite2-Country.mmdb"]
```
Comment (amitsaha) [inline .meta/transforms/geoip.toml]: We should remove the Country DB example file from here?

**Comment by amitsaha** [comment]
Thanks for the geoip docs @binarylogic - sorry I couldn't get to it in time.

**Comment by binarylogic** [comment]
Thanks @amitsaha, I've made those changes.

**Comment by amitsaha** [comment]
Thanks @binarylogic !

File: website/blog/2019-11-21-windows-support.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+It's important to point out that Vector's support for Windows is _first-class_.
+This is not a makeshift integration. Vector is written in [Rust][urls.rust],
+and Windows is a [tier 1 platform][urls.rust_tier_1_platform], meaning the same
+reliability and performance is brought to Windows as it is on Linux.
+
+## Fully-static without dependencies
+
+Beyond Rust's tier 1 support for Windows, Vector compiles to a single static
+binary. This makes installation as simple as copying the Vector binary onto
+your machine. There are no dependencies to install or environment changes
```
Comment (a-rodin) [inline website/blog/2019-11-21-windows-support.md]: I propose to clarify the last sentence like this:
> There are not additional DLL files to install or environment changes required.

**Pull request merged**